src/components/asset-list/RecyclerAssetList2/Claimable.tsx

```diff
@@ -25,11 +25,10 @@ export const Claimable = React.memo(function Claimable({ uniqueId, extendedState
   );
 
   const [claimable] = data;
+  if (!claimable) return null;
 
   const nativeDisplay = convertAmountToNativeDisplayWorklet(claimable.value.nativeAsset.amount, nativeCurrency, true);
```

Comment:
why isn't ts screaming about this? if it can be undefined I think the type is wrong somewhere

Reply:
i think it's coerced higher up but yeah likely a bigger issue